bitbots_dynamic_kick/launch/test.launch

```diff
@@ -2,6 +2,7 @@
   <include file="$(find bitbots_bringup)/launch/load_robot_description.launch" />
   <node name="rviz" pkg="rviz" type="rviz" output="screen" required="true"
     args="-d $(find wolfgang_description)/config/wolfgang.rviz" />
-  <node name="motor_goals_viz_helper" pkg="bitbots_bringup" type="motor_goals_viz_helper.py" args="--kick" />
-  <!--node name="dynamic_kick" pkg="bitbots_dynamic_kick" type="KickNode" output="screen" /-->
+  <node name="dynamic_kick" pkg="bitbots_dynamic_kick" type="KickNode" output="screen">
+        <remap from="/kick_motor_goals" to="/DynamixelController/command"/>
+  </node>
```

Comment:
Why?

This does essentially the same but adds duplication. Especially since you chose to use the viz helper in your `viz.launch`

Reply:
The motor_goals_viz_helper remaps to joint_states which includes a conversion to another message type. Therefore a remap is sufficient (and more efficient) here.

This can also be applied to your other comments.